server/controllers/apply/applicationsController.ts

```diff
@@ -13,17 +16,47 @@ export default class ApplicationsController {
     }
   }
 
-  new(): RequestHandler {
+  start(): RequestHandler {
     return (_req: Request, res: Response) => {
-      res.render('applications/new', { pageHeading: 'Apply for an Approved Premises (AP) placement' })
+      res.render('applications/start', {
+        pageHeading: 'Apply for an Approved Premises (AP) placement',
+      })
+    }
+  }
+
+  new(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { errors, errorSummary, userInput } = fetchErrorsAndUserInput(req)
+
+      const crnArr = req.flash('crn')
+
+      if (crnArr.length) {
+        const person = await this.personService.findByCrn(req.user.token, crnArr[0])
+
+        return res.render(`applications/confirm`, {
+          pageHeading: `Confirm ${person.name}'s details`,
+          ...person,
+          dateOfBirth: formatDateString(person.dateOfBirth),
+          errors,
+          errorSummary,
+          ...userInput,
+        })
+      }
+
+      return res.render('applications/new', {
+        pageHeading: "Enter the individual's CRN",
+        errors,
+        errorSummary,
+        ...userInput,
+      })
     }
   }
 
   create(): RequestHandler {
     return async (req: Request, res: Response) => {
       const uuid = await this.applicationService.createApplication(req.user.token)
```

Comment:
I think we'll want to pass the user to the application here too, but that can maybe come later